cmd/estuary-shuttle/data.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type Pin struct {
-	ID        uint      `gorm:"primarykey" json:"id"`
+	ID        uint      `gorm:"primaryKey" json:"id"`
```

Comment:
does this have any side effects on already created tables?

Reply:
it shouldn't, but will separate into another PR